src/scene/base.rs

```diff
@@ -342,6 +342,7 @@ pub struct Base {
     /// A set of custom properties that can hold almost any data. It can be used to set additional
     /// properties to scene nodes.
     pub properties: Vec<Property>,
+    frustum_culling: bool,
```

Comment:
You should also serialize this field. Go to `impl Visit for Base` and add somewhere at the end `let _ = self.frustum_culling.visit("FrustumCulling", visitor);`

Reply:
Does the visitor thing also handle de-serialization ?

Reply:
Visitor is a serializer+deserializer so yes :)